### PR TITLE
Draft tidyverse developer day save the date

### PR DESCRIPTION
**Pull request by batpigandme**

### CODE REVIEW

File: content/articles/2018-08-tidyverse-developer-day.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+### Save the date `r emo::ji("calendar")`
+
+On the Saturday following [rstudio::conf](https://www.rstudio.com/conference/) 
+(19 January), we'll be holding our first ever **tidyverse developer day**, and you're all invited! So, if you're interested, plan your travel accordingly.
```
Comment (jennybc) [inline content/articles/2018-08-tidyverse-developer-day.Rmarkdown]: If we have any concerns that interest will exceed capacity, we should find a different wording. Or decide the risk is one we're willing to take.

Comment (batpigandme) [reply content/articles/2018-08-tidyverse-developer-day.Rmarkdown]: I'll change the wording here, but I agree about the overall lack of detail that'll beg questions — I don't know the answers, but I'm happy to add any other info we have!

**Comment by batpigandme** [comment]
@hadley, do we know the answers to the questions that @jennybc mentioned, above? I was trying to err on the side of ambiguity in the absence of knowledge, but I feel like capacity and cost are pressing issues if someone's planning travel…

**Comment by hadley** [comment]
It will be in Austin and it will be free. I imagine it will be reasonably close to the conference venue, just because it's downtown and I expect that where the facilities are, but I don't want to promise anything yet. 

We should definitely create a community thread, and actively solicit questions and advice.

**Comment by batpigandme** [comment]
OK, [thread](https://community.rstudio.com/t/tidyverse-developer-day/13146) for questions on community will go live when the post does, I added an FAQ section below the body of the post, which we can edit and add to as we go..

**Pull request merged**